Works feature

hetrox8 opened the pull request:
This branch contains cards to be displayed . it appealing features are the cards 
-buttons
-and links
The main languages used to construct and style are html and css

V-Blaze commented on index.html:
```diff
@@ -207,6 +207,71 @@ <h2 class="headings">Tonics</h2>
 </div>
 
  <div class="card-works">
```
- [ ] You did great implementing the about section :+1: :+1:  but I think there are some improvements you need to make here before moving to the next milestone. 
- Following the best practices, your entire work section should be wrapped in a `section` tag. this is for semantics purposes as it helps in improving your website SEO. :+1: :smile:

V-Blaze replied:
- [ ] You did great implementing the WORK section :+1: :+1:  but I think there are some improvements you need to make here before moving to the next milestone. 
- Following the best practices, your entire work section should be wrapped in a `section` tag. this is for semantics purposes as it helps in improving your website SEO. :+1: :smile: 

- I think your project card is missing some items. Looking at the screenshot of the Figma design below, Your design is missing those DOT I circled, As part of this project requirement is to replicate the given design, I'll suggest you go through your design and update these. :+1: :+1: 
- The font color of those language lists ie `html CSS javascript` are not the same as the color from the design, I'll suggest you get the appropriate color and update yours. :+1: :smile: 
- Also I think your button needs a little bit of styling to match the figme design. and currently your button hover sate doesn't match the same background color and text color from the design, I'll suggest you get the correct color and update your CSS. :+1: :smile: :small_airplane: 

FIGMA DESIGN:
![Selection_033](https://user-images.githubusercontent.com/107632780/236300757-f0dd856b-dc1a-4ada-8ef4-d8a052a50d55.png)

YOUR DESIGN:
![Selection_034](https://user-images.githubusercontent.com/107632780/236300826-86abaf4d-9af5-4e8b-9a1e-6dedbf36565f.png)

V-Blaze commented on index.html:
```diff
@@ -207,6 +207,71 @@ <h2 class="headings">Tonics</h2>
 </div>
 
  <div class="card-works">
+  <header class="about-me"> 
+    <h2> About myself</h2>
+  </header>
```
- [ ] The `header` tag is used only to wrap the header section of the website. The header section is the topmost part of your app that has the website logo and the navigations, I'll suggest you remove this header tag here as you this is not your website header, :+1: :smile:

V-Blaze commented on index.html:
```diff
@@ -207,6 +207,71 @@ <h2 class="headings">Tonics</h2>
+  
+
+  </div>
+    <h1 class="resume">Get my resume</h1>
```
- [ ] The `Get my resume` is going to be used by your website visitors to get access to your resume, so I'll suggest you use a `button` or `a` here. :+1: 
- [ ] Also I noticed your design does not match the given Figma design, I'll suggest you go through the design one more time and make sure your design looks like the Figma design as that's part of the requirement for this milestone. :+1: 
- [ ] I'll suggest you implement the button states here `hover, active`. :+1:  


- YOUR DESIGN:
![Screenshot from 2023-05-04 19-24-23](https://user-images.githubusercontent.com/107632780/236296674-f580c8c7-9a56-4c1e-800f-fccd91c5e08d.png)

- FIGMA DESIGN:
![Screenshot from 2023-05-04 19-24-06](https://user-images.githubusercontent.com/107632780/236296757-091926b2-d0f8-40d4-98a0-2d805336ce4c.png)

V-Blaze commented on index.html:
```diff
@@ -207,6 +207,71 @@ <h2 class="headings">Tonics</h2>
+    <h1 class="resume">Get my resume</h1>
+  
+     <div class="langs">
+      <h1 class="lang1"> languages</h1>
```
- [ ] I'll suggest you use `h2 or h3` tag here as you are meant to have only 1 h1 in your webpage and that should be the main title. :+1: :smile: 

- [ ] As part of the requirement is to replicate the given design as much as possible, I think there are some improvements you need to make here. 
- - The text color of the text `languages` is different from the one in the given design, Please look into this.
- - The spacing between `Frameworks` and `Skills` in your design is more than the given design, I'll suggest you get the appropriate spacing and margin from the design and adjust yours accordingly.
- - Also you are missing the arrows at the right of the languages and frame works, I'll suggest you added this too to your design. :+1: :+1: 

- YOUR DESIGN:
![Selection_032](https://user-images.githubusercontent.com/107632780/236298537-8f1d627a-8e31-4ab2-9c34-c5bf87d9be9a.png)

- FIGMA DESIGN:
![Screenshot from 2023-05-04 19-31-32](https://user-images.githubusercontent.com/107632780/236298600-d4866404-4e0e-4e2a-a019-ca23258052b5.png)

V-Blaze commented on index.html:
```diff
@@ -207,6 +207,71 @@ <h2 class="headings">Tonics</h2>
+        <li> <a href="#"><img src="./images/Frame 11.png" alt=""></a></li>
+      </ul>
+      <div class="frameworks">
+      <h1 class="frameworks"> Frameworks</h1>
```
- [ ] I'll suggest you use `h2 or h3` tag here as you are meant to have only 1 h1 in your webpage and that should be the main title. :+1: :smile:

V-Blaze reviewed (changes requested):
## Hi @hetrox8 , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

![almost-there](https://user-images.githubusercontent.com/107632780/229242540-644ade4a-fe15-4f4c-a38b-6ded70cc4e8b.gif)


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Meaningful commit messages ✔
- Linters are green👋 
-  Readme looks professional✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

hetrox8 commented:
- wrapped the entire work section using the section tag
- removed the header from unnecessary places in the page .ie using it on the top most part of the page
- changed the Resume section from using a H1 to using a button tag. due to it's usage for visitors to access my resume
- implemented hover and active effects
- removed multiple h1  and h2 instead used only single h1 or h2
- changed the spaceing between the frameworks and skills section
- added down arrow and right arrow 
- changed color of  the text languages

divinecharlotte commented on index.html:
```diff
@@ -67,24 +67,21 @@ <h2>let's connect</h2>
     <!-- works -->
 
     
 <div class="works">
```
- [ ] Kindly as requested by the previous reviewer consider wrapping the entire section in a section tag not wrapping each work card in a section tag I think you've got the previous reviewer in a wrong way 
-  here is a hint :
```
<section class="works">
  
<div class="card-works">
 <div class="first-image">
  <img src="./images/Snapshoot Portfolio (1).png" alt="">
 </div>
  
 <h2 class="headings">Tonics</h2>

 <ul class="first-item">
   <li class="one"> <a href="#">Canopy</a> </li>
   <li class="other"> <a href="#">back end dev</a></li>
   <li class="other"> <a href="#">2015</a></li>
 </ul>

  <p class="text-1">
    A daily selection of privately <br>
     personalized reads; no accounts or sign-ups required.
  </p>

  <ul class="item-2">
    <li class="link-item"> <a href="#"></a> html</li>
    <li  class="link-item" > <a href="#">css</a></li>
    <li class="link-item"> <a href="#">javascript</a></li>

  </ul>

   <p class="see">
    <a href="#">See projects</a>
   </p>

  </div>
add other divs too .................................
</section>
```

this is for semantics purposes as it helps in improving your website SEO

divinecharlotte commented on index.html:
```diff
@@ -204,9 +203,70 @@ <h2 class="headings">Tonics</h2>
 
  <div class="card-works">
+  
+    <p class="abt-self"> About myself</p>
```
- [ ] kindly consider adding this in hx tag to meet the below requirement 
> The title (e.g., "About myself") uses a header tag hx
it might be h2,h3,...... not <header> tag

hetrox8 replied:
kindly be clear. "it might be h2,h3,...... not" what  is this  supposed to mean . not what ?

divinecharlotte commented on index.html:
```diff
@@ -204,9 +203,70 @@ <h2 class="headings">Tonics</h2>
-</div>
+</section >
 
  <div class="card-works">
```
- [ ] As requested by the previous reviewer kindly consider sticking to the Figma design as much as possible check on the image below and consider fixing the wrong design before you request another review doing so will make your project meet the design requirements
- your design 👇 
<img width="392" alt="Screen Shot 2023-05-05 at 18 34 30" src="https://user-images.githubusercontent.com/60146030/236516012-584721d9-201d-46a0-8682-f685829c382d.png">
  the borders and the top padding are not exactly like the one given below 

- Figma design 👇 
<img width="309" alt="Screen Shot 2023-05-05 at 18 37 18" src="https://user-images.githubusercontent.com/60146030/236516390-9e6261ed-a81e-4942-9fa2-2368ae8f06ba.png">

hetrox8 replied:
about the margin, i am to work on it.
here is your comment " Kindly as requested by the previous reviewer consider wrapping the entire section in a section tag not wrapping each work card in a section tag "  .. now at this part you are telling me that there should be borders, quick question how is that to happen if i am to have one card-works section  having the entire content..

hetrox8 replied:
i mean how just how

hetrox8 replied:
but you can clearly see the figma template each and every part has a border . now if i am to have a single section tag to contain all card will it be possible to have the exact replica . suppose i was to add divs inside the section tag to enable having card .then i believe it won't be able to produce the exact template . in other words what you are telling me is that section tags are to be used in a limit.. just one to contain the whole card

divinecharlotte commented on index.html:
```diff
@@ -204,9 +203,70 @@ <h2 class="headings">Tonics</h2>
+      Get my Resume
+    </button>
+ 
+    <h3 class="lang">languages <a href="#"> <img src="./images/down-arrow.svg" class="icon1" alt=""></a> </h3>  
+     <div class="langs">
+       
+      <ul class="languages">
+        <li> <a href="#"> <img src="./images/Frame 9.png" alt=""></a> </li>
+        <li> <a href="#"><img src="./images/Frame 10.png" alt=""></a> </li>
+        <li> <a href="#"><img src="./images/Frame 11.png" alt=""></a></li>
+      </ul>
+
+    </div>
+
+      <div class="framework">
+      <p class="frameworks"> Frameworks <img src="./images/right-arrow.svg"alt="" class="icon2"> </p>
+      <p class="skills">skills <img src="./images/right-arrow.svg" alt="" class="icon2"> </p>
+  
+      </div>
```
- [ ] [OPTIONAL] ☠️ (they are part of the requirement )kindly consider adding:

-  the   technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, each of them in a `li`
- The technology in each category(e.g., "JavasScript", "HTML" and "CSS") in a list `ul` and each of them a `li`
 as they are part of the requirement tho the previous reviewer missed it out  consider updating them to before requesting for another review to make your code professional

divinecharlotte reviewed (changes requested):
Hi @hetrox8     , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 
- Descriptive Pr 👍
- linters are passing 👍 
- Clear commit messages 👍 




 
![](https://media.giphy.com/media/tKyrc0NTlEcGWdThp4/giphy.gif)
## Required Changes ♻️

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

hetrox8 commented:
i am to work on the margin and borders  . yet you telling me i need to wrap
the entire works-card section as a single section containing  all cards

On Fri, May 5, 2023 at 7:59 PM suleiman gacheru ***@***.***>
wrote:

> 😑  un believable
>
> On Fri, May 5, 2023 at 7:45 PM divine charlotte ***@***.***>
> wrote:
>
>> ***@***.**** requested changes on this pull request.
>>
>> Hi @hetrox8 <https://github.com/hetrox8> ,
>>
>> Good job so far!
>> You still need to work on some issues to go to the next project, but you
>> are almost there!
>> To Highlight 💯
>>
>>    - Descriptive Pr 👍
>>    - linters are passing 👍
>>    - Clear commit messages 👍
>>
>>
>> <https://camo.githubusercontent.com/0c690ab997907a8c79afe7e695c891f6398cdbef93eaf8ce19f652ff321b54f8/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966>
>> Required Changes ♻️
>>
>> *Check the comments under the review.*
>> Optional suggestions
>>
>> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
>> the approval of this PR. However, I strongly recommend you to take them
>> into account as they can make your code better.*
>>
>> Cheers and Happy coding!👏👏👏
>>
>> Feel free to leave any questions or comments in the PR thread if
>> something is not 100% clear.
>> Please, remember to tag me in your question so I can receive the
>> notification.
>>
>> *Please, do not open a new Pull Request for re-reviews. You should use
>> the same Pull Request submitted for the first review, either valid or
>> invalid unless it is requested otherwise.*
>> ------------------------------
>>
>> *As described in the Code reviews limits policy
>> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
>> limited number of reviews per project (check the exact number in your
>> Dashboard). If you think that the code review was not fair, you can request
>> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>> ------------------------------
>>
>> In index.html
>> <https://github.com/hetrox8/portfolio1/pull/7#discussion_r1186284454>:
>>
>> >              <img src="./images/Vector.png" alt="vector">
>>
>>      </div>
>>    </section>
>>    </section>
>> -
>> -<br>
>> -
>>      <!-- works -->
>>
>>
>>  <div class="works">
>>
>>
>>    - Kindly as requested by the previous reviewer consider wrapping the
>>    entire section in a section tag not wrapping each work card in a section
>>    tag I think you've got the previous reviewer in a wrong way
>>    - here is a hint :
>>
>> <section class="works">
>>
>> <div class="card-works">
>>  <div class="first-image">
>>   <img src="./images/Snapshoot Portfolio (1).png" alt="">
>>  </div>
>>
>>  <h2 class="headings">Tonics</h2>
>>
>>  <ul class="first-item">
>>    <li class="one"> <a href="#">Canopy</a> </li>
>>    <li class="other"> <a href="#">back end dev</a></li>
>>    <li class="other"> <a href="#">2015</a></li>
>>  </ul>
>>
>>   <p class="text-1">
>>     A daily selection of privately <br>
>>      personalized reads; no accounts or sign-ups required.
>>   </p>
>>
>>   <ul class="item-2">
>>     <li class="link-item"> <a href="#"></a> html</li>
>>     <li  class="link-item" > <a href="#">css</a></li>
>>     <li class="link-item"> <a href="#">javascript</a></li>
>>
>>   </ul>
>>
>>    <p class="see">
>>     <a href="#">See projects</a>
>>    </p>
>>
>>   </div>
>> add other divs too .................................
>> </section>
>>
>> this is for semantics purposes as it helps in improving your website SEO
>> ------------------------------
>>
>> In index.html
>> <https://github.com/hetrox8/portfolio1/pull/7#discussion_r1186286897>:
>>
>> >
>>   <div class="card-works">
>> +
>> +    <p class="abt-self"> About myself</p>
>>
>>
>>    - kindly consider adding this in hx tag to meet the below requirement
>>
>> The title (e.g., "About myself") uses a header tag hx
>> it might be h2,h3,...... not
>> tag
>>
>> ------------------------------
>>
>> In index.html
>> <https://github.com/hetrox8/portfolio1/pull/7#discussion_r1186295634>:
>>
>> > @@ -204,9 +203,70 @@ <h2 class="headings">Tonics</h2>
>>    <a href="#">See projects</a>
>>  </p>
>>
>> -</div>
>> +</section >
>>
>>   <div class="card-works">
>>
>>
>>    - As requested by the previous reviewer kindly consider sticking to
>>    the Figma design as much as possible check on the image below and consider
>>    fixing the wrong design before you request another review doing so will
>>    make your project meet the design requirements
>>    - your design 👇
>>
>> [image: Screen Shot 2023-05-05 at 18 34 30]
>> <https://user-images.githubusercontent.com/60146030/236516012-584721d9-201d-46a0-8682-f685829c382d.png>
>> the borders and the top padding are not exactly like the one given below
>>
>>    - Figma design 👇
>>
>> [image: Screen Shot 2023-05-05 at 18 37 18]
>> <https://user-images.githubusercontent.com/60146030/236516390-9e6261ed-a81e-4942-9fa2-2368ae8f06ba.png>
>> ------------------------------
>>
>> In index.html
>> <https://github.com/hetrox8/portfolio1/pull/7#discussion_r1186300492>:
>>
>> > +    <h3 class="lang">languages <a href="#"> <img src="./images/down-arrow.svg" class="icon1" alt=""></a> </h3>
>> +     <div class="langs">
>> +
>> +      <ul class="languages">
>> +        <li> <a href="#"> <img src="./images/Frame 9.png" alt=""></a> </li>
>> +        <li> <a href="#"><img src="./images/Frame 10.png" alt=""></a> </li>
>> +        <li> <a href="#"><img src="./images/Frame 11.png" alt=""></a></li>
>> +      </ul>
>> +
>> +    </div>
>> +
>> +      <div class="framework">
>> +      <p class="frameworks"> Frameworks <img src="./images/right-arrow.svg"alt="" class="icon2"> </p>
>> +      <p class="skills">skills <img src="./images/right-arrow.svg" alt="" class="icon2"> </p>
>> +
>> +      </div>
>>
>>
>>    -
>>
>>    [OPTIONAL] ☠️ (they are part of the requirement )kindly consider
>>    adding:
>>    -
>>
>>    the technology categories ("Languages", "Frameworks" and "Skills") in
>>    a ul, each of them in a li
>>    -
>>
>>    The technology in each category(e.g., "JavasScript", "HTML" and
>>    "CSS") in a list ul and each of them a li
>>    as they are part of the requirement tho the previous reviewer missed
>>    it out consider updating them to before requesting for another review to
>>    make your code professional
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/hetrox8/portfolio1/pull/7#pullrequestreview-1415093585>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AZ6AOJCWIYLO6WAPHGXUOZTXEUVBNANCNFSM6AAAAAAXVWTC3Q>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

hetrox8 commented:
1. wrapped entire section in a section tag
2. changed the border and top padding  in the About area

NahnahAJ reviewed (approved):
## Project Status: APPROVED :green_circle: 
Hi @hetrox8 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
<img src="https://media4.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"/>

## Highlights
- All requested changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
